core/domain/user_services.py

```diff
@@ -1694,3 +1747,105 @@ def is_topic_manager(user_id):
+
+    Args:
+        user_id: str. The unique ID of the user.
+        language_code: str. The code of the language.
+
+    Returns:
+        bool. Whether the user can review translation suggestions in any
+        language or in the given language.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    reviewable_language_codes = (
+        user_community_rights.can_review_translation_in_languages)
+    if language_code is not None:
+        return language_code in reviewable_language_codes
+    else:
+        return True if reviewable_language_codes else None
+
+
+def can_review_voiceover_applications(user_id, language_code=None):
+    """Returns whether the user can review voiceover applications in any
+    language or in the given language.
+
+    NOTE: If the language_code is provided then this method will check whether
+    the user can review voiceover in the given language code else it will
+    check whether the user can review in any language.
+
+    Args:
+        user_id: str. The unique ID of the user.
+        language_code: str. The code of the language.
+
+    Returns:
+        bool. Whether the user can review voiceover applications in any language
+        or in the given language.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    reviewable_language_codes = (
+        user_community_rights.can_review_voiceover_in_languages)
+    if language_code is not None:
+        return language_code in reviewable_language_codes
+    else:
+        return True if reviewable_language_codes else None
+
+
+def can_review_question_suggestions(user_id):
+    """Checks whether the user can review question suggestion.
+
+    Args:
+        user_id: str. The unique ID of the user.
+
+    Returns:
+        bool. Whether the user can review question suggestion.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    return user_community_rights.can_review_questions
+
+
+def allow_user_review_translation_in_language(user_id, language_code):
+    """Allows the user with the given user id to review translation suggestions
+    in the given lanaguge_code.
+
+    Args:
+        user_id: str. The unique ID of the user.
+        language_code: str. The code of the language.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    user_community_rights.can_review_translation_in_languages.append(
+        language_code)
+    _save_user_community_rights(user_community_rights)
+
+
+def allow_user_review_voiceover_in_language(user_id, language_code):
+    """Allows the user with the given user id to review voiceover applications
+    in the given lanaguge_code.
+
+    Args:
+        user_id: str. The unique ID of the user.
+        language_code: str. The code of the language.
+    """
+    user_community_rights = get_user_community_rights(user_id)
+    user_community_rights.can_review_voiceover_in_languages.append(
```

Comment:
Ditto. Actually, also suggest always keeping the list unique and sorted, for cleanliness.